contracts/README.md

```diff
@@ -1,56 +1,74 @@
 # BLS Contract Wallet
+
 Lower-cost layer 2 transactions via a smart contract wallet.
 
+**Note:** _while these contracts have been formally audited, they are not deemed ready for production use._
```

Comment:
For our formal audit to be completed, we must address or accept all of its findings which is still in progress. I would recommend:

```suggestion
**Note:** _These contracts are in the process of being formally audited and are currently NOT recommended for production use_
```

Reply:
Amended